### ISSUE TITLE
Support for Razer Charging Pad Chroma

### ISSUE THREAD

**Issue by GeorgeeNico** [1kc/razer-macos#627]
Not sure if this is a new device; but any chance anyone could add support for the 10W fast charging pad by Razer?

**Comment by nhensh** [comment]
+ 1

**Comment by MateoWartelle** [comment]
Going to start and see if I can contribute to this.

**Comment by zangai** [comment]
@MateoWartelle Any progress on this? Anything I can do to help or take over from?

**Comment by xhi-nico** [comment]
I don't believe the original developer is active anymore. We might need someone to just start a new fork and take over this repo.

**Comment by MateoWartelle** [comment]
> from?

https://github.com/1kc/razer-macos/pull/741

I submitted a pull request but I don't think the maintainer is reviewing them.

**Comment by MateoWartelle** [comment]
> @MateoWartelle Any progress on this? Anything I can do to help or take over from?

https://github.com/1kc/razer-macos/pull/741

I submitted a pull request but I dont think the maintainer is reviewing.
